install_cluster.py

```diff
@@ -0,0 +1,260 @@
+    logger.info('Preparing PostgreSQL config files')
+    for node in instances_dict['postgresql']:
+        config_dict = get_dict_from_yaml(join(JUMP_HOST_DIR,
+                                              'postgresql_config.yaml'))
+        _write_certs_to_config(config_dict, 'postgresql_server', node.name)
+        config_dict['postgresql_server']['cluster']['nodes'] = \
+            _get_postgresql_cluster_members(instances_dict['postgresql'])
+
+        config_dict['manager']['private_ip'] = node.private_ip
+        config_dict['manager']['public_ip'] = node.public_ip
+        _create_config_file(config_dict, node.name)
+
+
+def _create_config_file(config_dict, node_name):
+    config_path = join(LOCAL_CONFIG_DIR, '{0}_config.yaml'.format(node_name))
+    with open(config_path, 'w') as config_file:
+        yaml.dump(config_dict, config_file)
+
+
+def _random_credential_generator():
+    return ''.join(random.choice(string.ascii_lowercase + string.digits)
+                   for _ in range(6))
+
+
+def _get_rabbitmq_cluster_members(rabbitmq_instances):
+    return {
+        rabbitmq_instances[j].name: {
+            'networks': {'default': rabbitmq_instances[j].private_ip}
+        } for j in range(len(rabbitmq_instances))}
+
+
+def _prepare_rabbitmq_config_files(instances_dict, rabbitmq_credentials):
+    logger.info('Preparing RabbitMQ config files')
+    first_rabbitmq = instances_dict['rabbitmq'][0]
+    rabbitmq_username, rabbitmq_password = rabbitmq_credentials
+    for i, node in enumerate(instances_dict['rabbitmq']):
+        config_dict = get_dict_from_yaml(join(JUMP_HOST_DIR,
+                                              'rabbitmq_config.yaml'))
+        config_dict['rabbitmq']['username'] = rabbitmq_username
+        config_dict['rabbitmq']['password'] = rabbitmq_password
+        config_dict['rabbitmq']['cluster_members'] = \
+            _get_rabbitmq_cluster_members(instances_dict['rabbitmq'])
+        _write_certs_to_config(config_dict, 'rabbitmq', node.name)
+        config_dict['rabbitmq']['nodename'] = node.name
+        if i != 0:
+            config_dict['rabbitmq']['join_cluster'] = first_rabbitmq.name
+
+        config_dict['manager']['private_ip'] = node.private_ip
+        config_dict['manager']['public_ip'] = node.public_ip
+        _create_config_file(config_dict, node.name)
+
+
+def _prepare_manager_config_files(instances_dict,
+                                  rabbitmq_credentials,
+                                  load_balancer_ip):
+    logger.info('Preparing Manager config files')
+    ca_path = join(REMOTE_CERTS_DIR, 'ca.pem')
+    for node in instances_dict['manager']:
+        config_dict = get_dict_from_yaml(join(JUMP_HOST_DIR,
+                                              'manager_config.yaml'))
+        config_dict['manager']['hostname'] = node.name
+        config_dict['manager']['private_ip'] = node.private_ip
+        config_dict['manager']['public_ip'] = node.public_ip
```

Comment:
Hmm... I think you're missing a manager.security.ssl = True?

Reply:
Nice catch, it's hardcoded in the manager_config.yaml